### ISSUE TITLE
please add you tube video quality option like 360p, 720p 

### ISSUE THREAD

**Issue by beauty-queen** [sarbagyastha/youtube_player_flutter#502]
please add you tube video quality option like 360p, 720p

**Comment by sha3rawi33** [comment]
Yes, we really need it. IDK why doesn't the official youtube API support it

**Comment by PcolBP** [comment]
Official API does support it but only on desktop browser. In youtube_player_iframe when you will enable desktopMode: true then button cog will be displayed. In mobile player when userAgent is null, player will adaptive quality to user speed network.
